README.md

```diff
@@ -14,3 +14,17 @@ part of the array each recursive call considers.
 Analyse the time complexity of your implementation and give a $\Theta$ bound for
 its worst-case runtime. Add your answer, including your reasoning, to this
 markdown file.
+
+## Answer
+
+Our outer loop in mergesort() runs log(n) times. This is because we are analyzing the input starting in pairs of two (our subarrays), and doubling each time. $2^0, 2^1, 2^2, ..., 2^k$, we stop when the subarrays are greater than the array length. $2^k >= n$ which equals $\Theta(logn)$.
+
+The inner loop has to run through the entire input each time it is called, this is taking $n$ time.
+
+I can now say that our mergesort() function has a time complexity of $\Theta(nlogn)$.
+
+mergeSubarrays() takes $\Theta(n^2)$ time to run, as it involves shifting elements inside the input, and there are two loops that go over $n$ elements in the worst case. 
+
+With this, I say that this program has a time complexity of $\Theta(n^3logn$).
```

Comment:
How do you get to the final complexity? It seems like you can't just multiply everything, similar to how we analyzed the "normal" mergesort.